kebechet/utils.py

```diff
@@ -43,8 +45,10 @@ def cwd(path: str):
 
 
 @contextmanager
-def cloned_repo(service_url: str, slug: str, **clone_kwargs):
+def cloned_repo(manager: Any, **clone_kwargs):
```

Comment:
```suggestion
def cloned_repo(manager: ManagerBase, **clone_kwargs):
```

Reply:
Its giving me an import error - 
```
(kebechet) [sai@sai-pc kebechet]$ ./kebechet-cli run-url -u https://github.com/thoth-station/source-management -s github
Traceback (most recent call last):
  File "./kebechet-cli", line 27, in <module>
    from kebechet.exception import WebhookPayloadError
  File "/home/sai/Downloads/thoth-station/kebechet/kebechet/__init__.py", line 4, in <module>
    from .managers import InfoManager  # noqa F401
  File "/home/sai/Downloads/thoth-station/kebechet/kebechet/managers/__init__.py", line 3, in <module>
    from .info import InfoManager
  File "/home/sai/Downloads/thoth-station/kebechet/kebechet/managers/info/__init__.py", line 3, in <module>
    from .info import InfoManager  # noqa F401
  File "/home/sai/Downloads/thoth-station/kebechet/kebechet/managers/info/info.py", line 23, in <module>
    from kebechet.managers.manager import ManagerBase
  File "/home/sai/Downloads/thoth-station/kebechet/kebechet/managers/manager.py", line 20, in <module>
    from kebechet.utils import APP_NAME
  File "/home/sai/Downloads/thoth-station/kebechet/kebechet/utils.py", line 26, in <module>
    from kebechet.managers import ManagerBase
ImportError: cannot import name 'ManagerBase'
```

Reply:
I added the import - `from kebechet.managers import ManagerBase`

Reply:
It looks like a cyclic import issue:

```
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from .manager import ManagerBase
```

then:

```
def cloned_repo(manager: "ManagerBase", **clone_kwargs):
```

Reply:
Thanks that does fix it :)